2023-04-09 17:12:46,117 INFO    MainThread:2695 [wandb_setup.py:_flush():68] Configure stats pid to 2695
2023-04-09 17:12:46,117 INFO    MainThread:2695 [wandb_setup.py:_flush():68] Loading settings from /Users/christoszacharopoulos/.config/wandb/settings
2023-04-09 17:12:46,117 INFO    MainThread:2695 [wandb_setup.py:_flush():68] Loading settings from /Users/christoszacharopoulos/projects/dynamic_risk/src/wandb/settings
2023-04-09 17:12:46,117 INFO    MainThread:2695 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-04-09 17:12:46,117 INFO    MainThread:2695 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'src/diagnostics.py', 'program': '/Users/christoszacharopoulos/projects/dynamic_risk/src/diagnostics.py'}
2023-04-09 17:12:46,117 INFO    MainThread:2695 [wandb_init.py:_log_setup():476] Logging user logs to /Users/christoszacharopoulos/projects/dynamic_risk/src/wandb/run-20230409_171246-74flzdwe/logs/debug.log
2023-04-09 17:12:46,117 INFO    MainThread:2695 [wandb_init.py:_log_setup():477] Logging internal logs to /Users/christoszacharopoulos/projects/dynamic_risk/src/wandb/run-20230409_171246-74flzdwe/logs/debug-internal.log
2023-04-09 17:12:46,117 INFO    MainThread:2695 [wandb_init.py:init():516] calling init triggers
2023-04-09 17:12:46,117 INFO    MainThread:2695 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2023-04-09 17:12:46,117 INFO    MainThread:2695 [wandb_init.py:init():535] re-initializing run, found existing run on stack: 2hge2ac7
2023-04-09 17:12:46,118 INFO    MainThread:2695 [wandb_run.py:_finish():1746] finishing run christonikos/dynamic_risk_assesment/2hge2ac7
2023-04-09 17:12:46,118 ERROR   MainThread:2695 [jupyter.py:save_history():440] Run pip install nbformat to save notebook history
2023-04-09 17:12:46,119 INFO    MainThread:2695 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2023-04-09 17:12:46,119 INFO    MainThread:2695 [wandb_init.py:_jupyter_teardown():408] cleaning up jupyter logic
2023-04-09 17:12:46,119 INFO    MainThread:2695 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2023-04-09 17:12:46,119 INFO    MainThread:2695 [wandb_run.py:_restore():1932] restore
2023-04-09 17:12:46,119 INFO    MainThread:2695 [wandb_run.py:_restore():1938] restore done
2023-04-09 17:12:52,342 INFO    MainThread:2695 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2023-04-09 17:12:52,343 INFO    MainThread:2695 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2023-04-09 17:12:52,351 INFO    MainThread:2695 [wandb_run.py:_footer_sync_info():3333] logging synced files
2023-04-09 17:12:52,356 INFO    MainThread:2695 [wandb_init.py:init():569] starting backend
2023-04-09 17:12:52,356 INFO    MainThread:2695 [wandb_init.py:init():573] setting up manager
2023-04-09 17:12:52,358 INFO    MainThread:2695 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-04-09 17:12:52,359 INFO    MainThread:2695 [wandb_init.py:init():580] backend started and connected
2023-04-09 17:12:52,360 INFO    MainThread:2695 [wandb_run.py:_label_probe_notebook():1113] probe notebook
2023-04-09 17:12:52,361 INFO    MainThread:2695 [wandb_run.py:_label_probe_notebook():1123] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-04-09 17:12:52,361 INFO    MainThread:2695 [wandb_init.py:init():658] updated telemetry
2023-04-09 17:12:52,391 INFO    MainThread:2695 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2023-04-09 17:12:54,249 INFO    MainThread:2695 [wandb_run.py:_on_init():2000] communicating current version
2023-04-09 17:12:54,599 INFO    MainThread:2695 [wandb_run.py:_on_init():2004] got version response upgrade_message: "wandb version 0.14.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-04-09 17:12:54,599 INFO    MainThread:2695 [wandb_init.py:init():728] starting run threads in backend
2023-04-09 17:12:55,839 INFO    MainThread:2695 [wandb_run.py:_console_start():1980] atexit reg
2023-04-09 17:12:55,840 INFO    MainThread:2695 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.WRAP_RAW
2023-04-09 17:12:55,840 INFO    MainThread:2695 [wandb_run.py:_redirect():1903] Wrapping output streams.
2023-04-09 17:12:55,840 INFO    MainThread:2695 [wandb_run.py:_redirect():1925] Redirects installed.
2023-04-09 17:12:55,840 INFO    MainThread:2695 [wandb_init.py:init():765] run started, returning control to user process
2023-04-09 17:13:30,803 INFO    MainThread:2695 [wandb_setup.py:_flush():68] Configure stats pid to 2695
2023-04-09 17:13:30,803 INFO    MainThread:2695 [wandb_setup.py:_flush():68] Loading settings from /Users/christoszacharopoulos/.config/wandb/settings
2023-04-09 17:13:30,803 INFO    MainThread:2695 [wandb_setup.py:_flush():68] Loading settings from /Users/christoszacharopoulos/projects/dynamic_risk/src/wandb/settings
2023-04-09 17:13:30,803 INFO    MainThread:2695 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-04-09 17:13:30,803 INFO    MainThread:2695 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'src/diagnostics.py', 'program': '/Users/christoszacharopoulos/projects/dynamic_risk/src/diagnostics.py'}
2023-04-09 17:13:30,803 INFO    MainThread:2695 [wandb_init.py:_log_setup():476] Logging user logs to /Users/christoszacharopoulos/projects/dynamic_risk/src/wandb/run-20230409_171330-205qhvz0/logs/debug.log
2023-04-09 17:13:30,803 INFO    MainThread:2695 [wandb_init.py:_log_setup():477] Logging internal logs to /Users/christoszacharopoulos/projects/dynamic_risk/src/wandb/run-20230409_171330-205qhvz0/logs/debug-internal.log
2023-04-09 17:13:30,803 INFO    MainThread:2695 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x15037dac0>
2023-04-09 17:13:30,804 INFO    MainThread:2695 [wandb_init.py:init():516] calling init triggers
2023-04-09 17:13:30,804 INFO    MainThread:2695 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2023-04-09 17:13:30,804 INFO    MainThread:2695 [wandb_init.py:init():535] re-initializing run, found existing run on stack: 74flzdwe
2023-04-09 17:13:30,804 INFO    MainThread:2695 [wandb_run.py:_finish():1746] finishing run christonikos/dynamic_risk_assesment/74flzdwe
2023-04-09 17:13:30,804 INFO    MainThread:2695 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2023-04-09 17:13:30,805 INFO    MainThread:2695 [wandb_run.py:_restore():1932] restore
2023-04-09 17:13:30,805 INFO    MainThread:2695 [wandb_run.py:_restore():1938] restore done
2023-04-09 17:13:36,522 INFO    MainThread:2695 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2023-04-09 17:13:36,523 INFO    MainThread:2695 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2023-04-09 17:13:36,528 INFO    MainThread:2695 [wandb_run.py:_footer_sync_info():3333] logging synced files
